packages/suite/src/components/firmware/FirmwareProgressBar.tsx

```diff
@@ -12,27 +12,34 @@ const Percentage = styled.div`
     width: 30px;
 `;
 
-const mapOperationToTransaltionId: Record<
-    NonNullable<FirmwareOperationStatus['operation']>,
-    TranslationKey
-> = {
-    installing: 'TR_INSTALLING',
-    validating: 'TR_VALIDATION',
-    restarting: 'TR_WAIT_FOR_REBOOT',
-    completed: 'TR_FIRMWARE_STATUS_INSTALLATION_COMPLETED',
+type FirmwareProgressBarProps = {
+    // In onboarding, the device is not initialized (no seed) yet, for example: there is no PIN during restart
+    isFirstInstallation: boolean;
```

Comment:
I think it would be better for this component to check `onboarding.isActive` rather than accept a prop.